src/directives/profile.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * @fileoverview Provides a directive used to insert an elevation profile chart
+ * in the DOM.
+ *
+ * Example:
+ *
+ * <div ngeo-profile="ctrl.profileData"
+ *      ngeo-profile-onhover="ctrl.onHover(point)"
+ *      ngeo-profile-onout="ctrl.onProfileOut()"></div>
```

Comment:
I think this is where we should document that `point` is a reference to an element of the input, user-provide data (`profileData` in the example).